src/Utils/SchemaExtender.php

```diff
@@ -615,13 +616,13 @@ static function (string $typeName) use ($schema) {
         $types = array_merge(
             // Iterate through all types, getting the type definition for each, ensuring
             // that any type not directly referenced by a field will get created.
-            array_map(static function ($type) {
+            array_map(static function (Type $type) : Type {
                 return static::extendNamedType($type);
-            }, array_values($schema->getTypeMap())),
+            }, $schema->getTypeMap()),
```

Comment:
didn't resetting of those array indexes through `array_values()` play some part?

Reply:
I don't see how, given the index is not used in the `array_map()`

Reply:
true, it's not `+`...